psst-gui/src/ui/album.rs

```diff
@@ -138,16 +139,29 @@ pub fn album_widget() -> impl Widget<WithCtx<Arc<Album>>> {
         .with_spacer(1.0)
         .with_child(album_artists)
         .with_spacer(1.0)
-        .with_child(album_date);
-
-    let album = Flex::row()
+        .with_child(album_date)
+        .fix_width(theme::grid(16.0));
+
+    let album_layout = if horizontal {
+        Flex::column()
+            .with_child(album_cover)
+            .with_default_spacer()
+            .with_child(album_info)
+            .fix_width(theme::grid(16.0))
+            .padding_horizontal(theme::grid(1.0))
+            .align_left()
+    } else {
+        Flex::row()
         .with_child(album_cover)
         .with_default_spacer()
         .with_flex_child(album_info, 1.0)
-        .padding(theme::grid(1.0))
-        .lens(Ctx::data());
+        .with_spacer(1.0)
+        .align_left()
+    };
```

Comment:
Was there a reason to change this on the default album_widget?

Reply:
Ah no there wasn't this was a slip up!